Makefile

```diff
@@ -541,18 +542,23 @@ lint-spell:
 
 lint: lint-js lint-sass lint-cpp lint-html lint-spell
 
-ELECTRON_MOCHA_OPTIONS=--recursive --reporter spec
+MOCHA_OPTIONS=--recursive --reporter spec
+
+test-spectron:
+	$(NPX) env \
+		ETCHER_SPECTRON_ENTRYPOINT=$(BUILD_DIRECTORY_PARENT)/node_modules/.bin/electron \
```

Comment:
Could we have
```
ETCHER_SPECTRON_ENTRYPOINT ?= $(BUILD_DIRECTORY_PARENT)/node_modules/.bin/electron
```
up near the top of the Makefile, and then change this line to:
```
ETCHER_SPECTRON_ENTRYPOINT=$(ETCHER_SPECTRON_ENTRYPOINT) \
```
?

That should allow us to then manually run e.g.
```
make ETCHER_SPECTRON_ENTRYPOINT=$(which etcher-electron) test-spectron
```
on the  command line, to run the spectron tests against the .deb installed version of Etcher :-)

Reply:
Sounds good, done!